vsprojects/vcxproj/grpc_csharp_ext/grpc_csharp_ext.vcxproj

```diff
@@ -101,6 +101,7 @@
       <SubSystem>Windows</SubSystem>
       <GenerateDebugInformation Condition="!$(Jenkins)">true</GenerateDebugInformation>
       <GenerateDebugInformation Condition="$(Jenkins)">false</GenerateDebugInformation>
+      <OutputFile>$(OutDir)$(TargetName)-x64$(TargetExt)</OutputFile>
```

Comment:
Understood.  I was too focused on making it work and did not look more closely at this.

I will also separate the different functionality into separate pull requests.  Thank you for taking the time to look into this.